fish_prompt.fish

```diff
@@ -850,14 +850,35 @@ function __bobthefish_prompt_desk -S -d 'Display current desk environment'
     set_color normal
 end
 
-function __bobthefish_prompt_nvm -S -d 'Display current node version through NVM'
-    [ "$theme_display_nvm" = 'yes' -a -n "$NVM_DIR" ]
+function __bobthefish_prompt_nodejs -S -d 'Display current Node.js version'
+    [ "$theme_display_nodejs" = 'yes' -o "$theme_display_nvm" = 'yes' ]
     or return
```

Comment:
I left the existing configuration setting `theme_display_nvm` here for anyone who is currently using it. Let me know if this is okay, or if you want me to take another approach. :smiley: